packages/components/future/README.md

```diff
@@ -0,0 +1,27 @@
+# Entrypoint: future
+
+This is directory is here to redirect any imports from `@kaizen/components/future` to the correct distribution of compiled code to allow for treeshaking of future components.
+
+More details: [https://github.com/theKashey/multiple-entry-points-example](https://github.com/theKashey/multiple-entry-points-example)
+
+## Contribution
+
+All "future" components are to be added to the `src/__future__` directory.
+
+### What is a "future" component?
+
+These are components which are scheduled to replace an already existing component. **It is not** to house unfinished components.
```

Comment:
So where do unfinished components go?

Reply:
We shouldn't have "unfinished" components in the library ie. components that don't match definition of done or components we aren't happy for people to start using yet.

Reply:
Then how do we do iterative work on it? :'( You want 6 month old branches with 5000 lines of code? 😭

Reply:
Well scoped work with clear definition of done, so we can release something as soon as possible 😄 -- edge cases can go ahead and be added, but imo anything taking 6mo and 5k lines is a flag.